modules/check_in/app/services/v2/chip/client.rb

```diff
@@ -149,11 +149,7 @@ def refresh_precheckin(token:)
         log_message_to_sentry(e.original_body, :error,
                               { uuid: check_in_session.uuid },
                               { external_service: service_name, team: 'check-in' })
-        if Flipper.enabled?('check_in_experience_chip_500_error_mapping_enabled')
-          raise e
-        else
-          Faraday::Response.new(body: e.original_body, status: e.original_status)
-        end
+        raise e
```

Comment:
Where does this error get raised?

Reply:
It will get raised all the up to the [controller](https://github.com/department-of-veterans-affairs/vets-api/blob/ae148b2fb2a1804b0e44c42b3c2fc9d18cedb0b9/modules/check_in/app/controllers/check_in/v2/sessions_controller.rb#L14) & front end will handle the error response to display appropriate page for the user

Reply:
I see. Thanks for the info!